pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -282,6 +283,7 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
         berry_id = ITEM_RAZZBERRY
         maximum_ball = ITEM_ULTRABALL if is_vip else ITEM_GREATBALL
         ideal_catch_rate_before_throw = 0.9 if is_vip else 0.35
+        hit_rate = self.config.catch_throw_parameters_hit_rate
```

Comment:
Tried putting in these changes manually into current dev.

Get error:
  File "~/PokemonGo-Bot/pokemongo_bot/cell_workers/pokemon_catch_worker.py", line 319, in _do_catch
    hit_rate = self.config.catch_throw_parameters_hit_rate
AttributeError: 'dict' object has no attribute 'catch_throw_parameters_hit_rate'